projects/plugins/boost/compatibility/elementor.php

```diff
@@ -19,7 +20,17 @@ function exclude_elementor_library_custom_post_type( $post_types ) {
 		unset( $post_types[ Source_Local::CPT ] );
 	}
 
+	// Elementor's landing pages are broken. See https://github.com/elementor/elementor/issues/16244
+	if ( isset( $post_types[ Module::CPT ] ) ) {
+		unset( $post_types[ Module::CPT ] );
```

Comment:
Doesn't it scream and shout when elementor is not installed and the classes `Module` and `Source_Local` is missing?

Reply:
No, because there is a class exists check before requiring the file.